moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/plugins/BeanValidationPlugin.java

```diff
@@ -756,7 +752,7 @@ else if (annotationName.equals(ANNOTATION_DECIMALMAX.fullName()))
      */
     private boolean isMoreSpecificBoundary(JFieldVar fieldVar, String boundaryValue, JAnnotationUse annotationUse,
                                            boolean xorComplement) {
-        String existingBoundaryValue = getExistingBoundaryValue(annotationUse);
+        String existingBoundaryValue = annotationUse.getAnnotationMembers().get(VALUE).toString();
```

Comment:
> execution is delegated to com.sun.xml.bind:jaxb-xjc library

yes

> => stronger integration/dependency?

this is there since 2.6 (maybe 2.5.3), it was only hidden behind reflection (as many other calls to codemodel), so now the dependency is immediately visible and easier to maintain